hybrasyl/Messaging/Admin.cs

```diff
@@ -0,0 +1,410 @@
+﻿using Hybrasyl.Creatures;
+using Hybrasyl.Objects;
+using Hybrasyl.Scripting;
+using System;
+using System.Net;
+
+namespace Hybrasyl.Messaging
+{
+    // Various admin commands are implemented here.
+
+    class SummonCommand : ChatCommand
+    {
+        public new static string Command = "summon";
+        public new static string ArgumentText = "<string playerName>";
+        public new static string HelpText = "Summon a specified player to you";
+        public new static bool Privileged = true;
+
+        public new static ChatCommandResult Run(User user, params string[] args)
+        {
+            if (!Game.World.WorldData.ContainsKey<User>(args[0]))
+                return Fail($"User {args[0]} not logged in");
+
+            var target = Game.World.WorldData.Get<User>(args[0]);
+
+            if (target.IsExempt)
+                return Fail($"User {user.Name} is exempt from your meddling");
+
+            target.Teleport(user.Map.Id, user.MapX, user.MapY);
```

Comment:
Should be user.X and user.Y

Reply:
MapX and MapY are correct.

Reply:
updated to use `Location` for consistency, we may wish to do something about the fact that `Creature` doesn't use the same pattern

Reply:
yes, this should be standardized across `VisibleObject` base and not be restricted to `User`

Reply:
OK, will refactor.